README.md

```diff
@@ -10,46 +10,93 @@ This project uses [uv](https://github.com/astral-sh/uv).
 
 Thunderbird Accounts is a Django 5.1 application. Please read up on Django before diving in.
 
-## Setup / Running
+Docker is the recommended way to run the server, though you have the option to run it locally.
 
-### Installing
 
-Ensure that uv is installed, and then run:
+## Shared setup
+
+Whether you're running the server via Docker or locally, you'll need to set up your `.env` file and copy the example Stalwart config.
+
+### Create the .env file
+
+Start from the `.env.example` file to create your dev and test `.env` files:
+
 ```shell
-uv sync
+cp .env.example .env
+cp .env.example .env.test
 ```
 
-### Using the example config for Stalwart
+In `.env.test`, set `APP_ENV=test`
 
-Note: Do not use this for production environments.
+#### Get the `FXA_CLIENT_ID` and `FXA_SECRET`
 
-Copy config.toml.example to `mail/etc/config.toml` before first run.
+These can be found in the Services 1password vault in the item named "FXA stage application keys for TB Accounts".
 
-The default admin password should be `accounts`. 
+#### Generate `SECRET` keys
 
-### Creating a local SuperUser
+Generate a different, random secrets for `SECRET_KEY` and `LOGIN_CODE_SECRET` using this command:
 
-Run the following and follow the terminal prompts:
+```shell
+openssl rand -hex 16
+```
+
+### Use the example config for Stalwart
+
+Note: Do not use this for production environments.
+
+Copy config.toml.example to `mail/etc/config.toml` before first run.
 
 ```shell
-./manage.py createsuperuser
+mkdir -p mail/etc && cp config.toml.example mail/etc/config.toml
 ```
 
-### Running in docker (recommended)
+The default admin password should be `accounts`.
+
+
+## Running the server in Docker (recommended)
+
+
+### Start the Docker containers
 
 ```shell
 docker-compose up --build -V
 ```
 
 A server will be available at [http://localhost:8087/](http://localhost:8087/)
 
-### Running local dev server
+### Generate an FXA Encryption Secret
+
+
+Generate one with this command:
 
 ```shell
-./manage.py runserver 0.0.0.0:8087
+docker compose exec backend uv run manage.py generate_key
```

Comment:
I think we can probably use this value for the other secret values too. To keep it simple.

Reply:
Gotcha - I can:
* nix the `openssl` commands
* have them use `generate_key` as part of the "Shared setup"
* use that key for all three `*SECRET*` env vars

Reply:
Sounds good to me!